types/ember-modal-dialog/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": false, // 'ember' types do not compile when true
```

Comment:
please remove the comments, it is throwing the linter off.

Reply:
Oops! Silly me. Thanks for the catch.